rules/okta_rules/okta_idp_signin.py

```diff
@@ -0,0 +1,17 @@
+from panther_base_helpers import okta_alert_context
+
+
+def rule(event):
+    return event.get("eventType") == "user.authentication.auth_via_IDP"
```

Comment:
Same comment re: default.

Reply:
I agree with default strings for human readable things like title, but for boolean logic if 'eventType' doesn't exist, then default value is None and the rule returns False which is expected behavior.